configs/opencv-ovms/scripts/grpc_python.py

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+import sys
+sys.path.append("/model_server/demos/common/python")
+
+import argparse
+import datetime
+import cv2
+import grpc
+from client_utils import print_statistics
+from tritonclient.grpc import service_pb2, service_pb2_grpc
+from grpc_postprocess import *
+
+def openInputSrc(input_src):
+    # OpenCV RTSP Stream
+    stream = cv2.VideoCapture(input_src)
+    if not stream.isOpened():
+        print('Cannot open RTSP stream')
```

Comment:
add debugging logs about what `input_src` could not be opened

Reply:
Adding more log data